app/App/Panel/Main/Account/Requests/TransactionRequest/TxApproval/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
```

Comment:
if this is going to be a factory component I think we should at least make it have a standard API instead of taking props that are specific to only one type of approval (eg `allowOtherChain`)

in this way we can make the props API something more standard like `onApprove, onDecline` and have this component provide defaults if those arent passed in since a lot of the approvals will be using the same mechanism for decline and approve

this also allows us to centralize these methods and only have to change them in one place if change, say, the RPC interface